src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -3734,9 +3734,11 @@ public void M()
 {
     public void M()
     {
-        var s2 = string.Replace($""hello"", ""world"");
+        var s2 = string.Replace((string)$""hello"", ""world"");
     }
 }");
+
+            // PROTOTYPE: why? is it a C# 8.0 bug?
```

Comment:
This may be the result of overload resolution changes in 7.3. See corresponding changes in https://github.com/dotnet/roslyn/blob/features/NullableReferenceTypes/src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs#L3722